### ISSUE TITLE
Varimathras Necrotic Embrace is Buggy HC / Mythic

### ISSUE THREAD

**Issue by Dark-Sider** [BigWigsMods/BigWigs#487]
### What steps will reproduce the problem?  
1.  Get Necrotic Embrace on HC / Mythic difficulties when fighting varimathras
2.  Say Bubble won't show up (although configured), emphasize text wont show up
3.  wipe.

### What version of BigWigs are you using? (Stating 'latest' is not useful)
95.x (0402faa)

### Do you have an error log of what happened?
no errors are reported 

### Any additional information? (example: WoW language if not English) (Attach screenshots here if available, do not link externally)
English-UK client. Worked before latest update just fine. However I'm not sure that what my last version was - probably pre 94 (before latest vari changes in the changelog)

**Comment by funkydude** [comment]
Does this always happen? Was it only during a screwup where it was spreading a lot?

**Comment by Justw8** [comment]
We have been unable to reproduce this, is it possible for you to record this happening perhaps? This could help us identify the issue.

**Comment by Justw8** [comment]
https://github.com/BigWigsMods/BigWigs/commit/ba9f4a1872e2f8373d54dc3e9e6c9d3a6f9455f0 Pushed a fix for this, please let us know if you still have the problem.

**Comment by Dark-Sider** [comment]
Thx, I'll test this the next I'll pull vari on HC or Myth

**Comment by Justw8** [comment]
This fix only changed the text of the warning, so it doesn't really affect the say message at all.
However, there might be an additional issue with having the option "Only when on Me" enabled. 
Do you happen to have that enabled?

**Comment by Dark-Sider** [comment]
Yes I have the option "only when on me" enabled as I'm playing DD. However the say did not work with or without the "only when on me" enabled

**Comment by Justw8** [comment]
I recommend trying it with today's fix and having "Only on Me" disabled and let us know if that functions correctly.

We will work on a fix for the Only on Me function, but it requires some discussion between the devs.

**Comment by Justw8** [comment]
This should allow say messages to function when "Only when on Me" is enabled. 

Keep in mind however, only 2 players will have say messages every debuff jump. So if it jumps to more than 2 players, not everyone will have a countdown and/or warning (and usually it means you are wiping)

**Comment by Dark-Sider** [comment]
I can confirm that say and "only when on me" works with both fixes in HC. No mythic raid yet.
